src/Kafka/KafkaInstanceDrawer/KafkaDetailsTab.stories.tsx

```diff
@@ -17,20 +17,45 @@ const Template: ComponentStory<typeof KafkaDetailsTab> = (args) => (
   <KafkaDetailsTab {...args} />
 );
 
-export const NormalInstance = Template.bind({});
-NormalInstance.args = {};
+const instanceInfo = {
+  ingress: "up to 50 MiB/second",
+  egress: "up to 50 MiB/second",
+  storage: "up to 1000 GiB",
+  maxPartitions: "up to 1500",
+  connections: "up to 2000",
+  connectionRate: "up to 100 connections/seconds",
+  messageSize: "up to 1 MiB",
```

Comment:
The passed value should be just the numeric part. The string is responsibility of the UI and will have to be handled by the i18n framework

Reply:
This data will be come through api. Not sure in which format api will return the data. If we are sure that api will return the data in numeric format  then  we can do changes now or maybe later we can change it when api will be ready. Anyways these changes are available storybook components  only, not in production code.

Reply:
It will be numbers